server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -984,7 +984,7 @@ synchronized void completeClose(boolean saveState, boolean completeClose) throws
             activeScans.size());
         this.wait(50);
       } catch (InterruptedException e) {
-        log.error(e.toString());
+        log.error("{}", e, e);
```

Comment:
```suggestion
        log.error("Interrupted waiting to completeClose for extent {}, {}", extent, e, e);
```

Reply:
This is fine with me, it provides more context.